tests/kafkatest/services/performance/consumer_performance.py

```diff
@@ -127,15 +135,31 @@ def start_cmd(self, node):
         cmd += " /opt/%s/bin/kafka-consumer-perf-test.sh" % kafka_dir(node)
         for key, value in self.args.items():
             cmd += " --%s %s" % (key, value)
-        cmd += " --consumer.config %s" % ConsumerPerformanceService.CONFIG_FILE
 
-        for key, value in self.settings.items():
-            cmd += " %s=%s" % (str(key), str(value))
+        if node.version > LATEST_0_8_2:
+            # This is only used for security settings
+            cmd += " --consumer.config %s" % ConsumerPerformanceService.CONFIG_FILE
```

Comment:
Yeah, that's why I only asked about the two lines copying the `settings`. So, I was going to add it back after this `if` block. Is that correct?